zou/app/services/time_spents_service.py

```diff
@@ -149,6 +149,19 @@ def get_time_spents(person_id, date):
     return fields.serialize_list(time_spents)
 
 
+def get_time_spent(person_id, task_id, date):
+    """
+    Return time spents for given person, task and date.
+    """
+    try:
+        time_spent = TimeSpent.query.filter_by(
```

Comment:
you should use plural for the variable name: `time_spents`

Reply:
I use first() of all() so it's only a "time_spent"